code/modules/recycling/sortingmachinery.dm

```diff
@@ -185,6 +188,15 @@
 			wrapped_item.AddComponent(/datum/component/pricetag, sticker.payments_acc, sticker.cut_multiplier)
 		update_appearance()
 
+	if(istype(item, /obj/item/boxcutter))
+		if( (item.icon_state == "boxcutter_on"))
+			unwrap_contents(user)
+			post_unwrap_contents(user, rip_open = FALSE)
+			return
+		else
+			balloon_alert(user, span_warning("The boxcutter isn't active!"))
```

Comment:
```suggestion
			balloon_alert(user, "prime the boxcutter!")
```

Balloon alerts are not styled the same way as chat messages, and the wording on this wasn't very boxcutter-y. Feel free to replace prime with arm or ready.